src/taglib/taglib-loader/Tag.js

```diff
@@ -107,6 +112,12 @@ class Tag {
                 ) {
                     attr.targetProperty = null;
                 } else if (!attr.targetProperty) {
+                    complain(
+                        "@* target-property property required. The recommended value is to set target-property to null.",
```

Comment:
I would make the complain something like:

```
The default "targetProperty" for "@*" attribute definitions is changing from "*" to "null" (merged in with the rest of the input) in a future Marko release. In order to avoid an issue upgrading, please explicitly define the "targetProperty".
```